### DISCUSSION TITLE
Help with Local Development and Deployment of Generic Editor

### DISCUSSION THREAD

**Discussion by rajnishdargan** [Sunbird-Knowlg/Community#220] [Q&A]
Hello,

For our local development related to integrating the generic editor into our application, we are accessing the generic-editor, content-plugins, and other dependencies from Sunbird SaaS by proxying the endpoint like this:

http://localhost:3000/editors/generic-editor/index.html

This call is redirected to:

https://dev.sunbirdsaas.com/generic-editor/index.html

We need to understand how these components (e.g., generic-editor) are deployed to the Sunbird portal. We will not be using https://dev.sunbirdsaas.com, but instead, we will be using a separate portal where we plan to utilize the latest version of the generic-editor. Additionally, we might make changes related to AWS cloud storage and deploy it.

I have searched for the deployment steps but could not find relevant details in the Knowlg microsite. I would also like to know whether it is possible to point to the editors locally by following specific build steps. Specifically, can we move the content-plugins, generic-editor, and other dependencies to a local folder and use them directly?

I believe the dist folder of the editor is what gets deployed to cloud storage. Please correct me if I’m wrong. I attempted to build the editor using Node.js 12, but the build failed.

Could you please guide me on how we can use the generic-editor locally for integration purposes and how we can deploy it successfully?

@pallakartheekreddy @HarishGangula @rajeevsathish @vinukumar-vs @vaibhavbhuva
@coolbung @itsvick @AbhilashKD

**Comment by pallakartheekreddy** [comment]
@rajnishdargan lets get into a call to check this issue.
